File: 2025_03/[코딩 기초 트레이닝] 1로 만들기/1로 만들기.py
def solution(num_list):
    answer = 0

    for n in num_list:
        # 홀수일 때 1빼고 2로 나누기 -> 몫만 갖고싶다는 의미
        while n != 1:
            n //= 2
            answer += 1

    return answer

File: 2025_03/프로그래머스 - 코딩 기초 트레이닝/[코딩 기초 트레이닝] 1로 만들기/1로 만들기.js
function solution(num_list) {
  let answer = 0;
  num_list.forEach((value) => {
    while(value !== 1){
      // 홀수일때 1 빼고 2로 나누기 -> 그냥 홀수 2로 나누고 몫 구하기랑 같음
      value = Math.floor(value / 2)
      answer++;
    }
  })
  return answer;
}